README.md

```diff
@@ -14,7 +12,7 @@ This service is API only, and should be used together with [Envelop-UI](https://
 
 This service depends on a couple of services:
 
-- Node.js (>= 18)
+- Node.js (~ 16)
```

Comment:
It should be the same as here https://github.com/snapshot-labs/envelop/blob/49879f62bcd492f3832de3bd839573129dbe903e/package.json#L25
or should be >=16 right? `~` may mean we support 15.x too

Reply:
[it doesn't](https://vibhas1892.medium.com/difference-between-tilde-and-caret-in-package-json-76701e514445#:~:text=npm%20uses%20the%20tilde%20(~),means%20to%20install%20version%201.0.). In any case you are right